ssh/tailssh/incubator.go

```diff
@@ -0,0 +1,281 @@
+// Copyright (c) 2022 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
```

Comment:
Maybe a comment up here explaining what an "incubator" is?  (the middle of the three processes, running as root?)